src/gen_experiments/pdes.py

```diff
@@ -0,0 +1,103 @@
+import numpy as np
+from pysindy.differentiation import SpectralDerivative
+from .utils import (
+    gen_pde_data,
+    compare_coefficient_plots,
+    plot_pde_training_data,
+    coeff_metrics,
+    integration_metrics,
+    unionize_coeff_matrices,
+    _make_model,
+)
+
+name = "pdes"
+
+def diffuse1D(t, u, dx, nx):
+    u = np.reshape(u, nx)
+    u[0] = 0
+    u[-1] = 0
+    uxx = SpectralDerivative(d=2, axis=0)._differentiate(u, dx)
+    return np.reshape(uxx, nx)
+
+pde_setup = {
+    "diffuse1D": {
+        "rhsfunc": {
+            "func": diffuse1D,
+            "dimension": 1
+        },
+        "input_features": ["u"],
+        "initial_condition": 10*np.exp(-(np.arange(0, 10, 0.1)-5)**2/2),
+        "spatial_args": [0.1, 100],
+        "time_args": [0.1, 10],
+        "coeff_true": [
+            {"u_11": 1}
+        ],
+        "spatial_grid": np.arange(0, 10, 0.1)
+    },
+}
+
+def run(
+    seed: float,
+    /,
+    group: str,
+    diff_params: dict,
+    feat_params: dict,
+    opt_params: dict,
+    display: bool = True,
+    return_all: bool = True,
```

Comment:
default to `False` please, this is used to turn on a much larger return dictionary when used inside another experiment that builds composite plots (e.g. gridsearch).  This is what's causing the much larger output.